backend/src/service/attendance/attendance_service.py

```diff
@@ -41,8 +41,9 @@ def get_attendance_by_id(self, attendance_id: int) -> Optional[List]:
         """
         try:
             self.logger.info("Get attendance info by attendance_id:{}".format(attendance_id))
-            return [attendance.attendance_dict(school, student) for attendance, school, student in
+            attendance = [attendance.attendance_dict(school, student) for attendance, school, student in
```

Comment:
Can we please return only one record from the model itself? Because `get_*_by_id` methods are just trying to find one object anyway. 

Please fix AttendanceModel.get_attendance_by_id method to just return one AttendanceModel object.